src/docs/docs/getting-started/environment-setup.md

```diff
@@ -136,7 +136,11 @@ newgrp docker
 - Disable docker on startup: `sudo systemctl disable docker`
 
 11. Verify your installation by running `docker run hello-world`. This should print a hello world paragraph that includes a confirmation that Docker is working on your system.
-    _NOTE: This may cause errors if you have already tried to run docker before. If you get errors then run the following commands to reset it:_
+    :::note
+
+    This may cause errors if you have already tried to run docker before. If you get errors then run the following commands to reset it
+
+    :::
```

Comment:
Remove tabs here, because they make the note slightly off to the right and it doesn't align.